website/docs/reference/resource-configs/type.md

```diff
@@ -2,30 +2,49 @@
 title: type
 sidebar_label: "type"
 id: type
+description: "The type config specifies the type of user-defined function you're creating."
 ---
 <VersionCallout version="1.11" /> 
 
 <File name='functions/<filename>.yml'>
 
 ```yml
 functions:
-  - name: <function name>
+  - name: function_name
     config:
-      type: scalar | aggregate | table  # aggregate and table coming soon
+      type: scalar | aggregate | table  # scalar & aggregate partially supported, table not supported
```

Comment:
I don't think we should include `table` here if it's not supported and since the note below states we're "considering" it and it's not officially coming soon